Added Timezones and Daylight Saving Time (DST) section (#4398)
Particular/docs.particular.net: nservicebus/sagas/timeouts.md

include: non-null-task

## Timezones and Daylight Saving Time (DST)

> Requesting a timeout can be done by passing either a `DateTime` or a `TimeSpan`. Passing a `DateTime` requires its `Kind` property to be set. If the timeouts needs to be triggered at a specific hour instead of a day then make sure that the time calculation is daylight saving time (DST) aware as clocks could shift a hour back or forth. Correct timezone and daylight saving time conversion can be done via the [`TimeZoneInfo.ConvertTime`](https://docs.microsoft.com/en-us/dotnet/api/system.timezoneinfo.converttime) API.

Reviewer: ```suggestion
A timeout may be requested specifying either a `DateTime` or `TimeSpan`. When specifying a `DateTime`, the `Kind` property must be set. If the timeout specifies a time of day, the calculation must take into account any change to or from DST. Timezone and DST conversion may be done using [`TimeZoneInfo.ConvertTime`](https://docs.microsoft.com/en-us/dotnet/api/system.timezoneinfo.converttime).
```

> NOTE: Keep in mind that DST / timezone info can change in the future for timeouts that are already set. Have sagas react to such changes if business logic is dependant on this.

Reviewer: ```suggestion
NOTE: Timezone and DST information may change in the future, for timeouts that are already set. A saga containing business logic which is dependent on such changes must react to those changes appropriately.
```

> NOTE: Keep in mind that DST / timezone info can change in the future for timeouts that are already set. Please keep in mind to have sagas react to such changes if business logic is dependant on this.

Reviewer 2: ```suggestion
NOTE: Keep in mind that DST / timezone info can change in the future for timeouts that are already set. Have sagas react to such changes if business logic is dependant on this.
```
